src/utils.js

```diff
@@ -171,25 +174,23 @@ async function main() {
 
         console.log(`Restored account ${keyPair.meta.name} ${keyPair.address} ...`);
 
-        const account = new Account(keyPair);
-        await account.init([mangataHelper]);
-        account.print();
-
-        const { pair } = account;
         const targetSellAmount = 100;
-        const minExpectedBuyAmount = 150;
+        // const minExpectedBuyAmount = 150;
```

Comment:
What’s up with this line of code comment? Do we still need it?

Reply:
We don't need the variable.
Removed.